test/utils/windowUtilsTests.ts

```diff
@@ -1,8 +1,52 @@
 ///<reference path="../testReference.ts" />
 
 describe("Utils.Window", () => {
-  describe("deprecated()", () => {
+  describe("warn()", () => {
+    const runTest = console != null ? it : it.skip;
+
+    runTest("uses console.warn if available, console.log otherwise", () => {
+      const originalWindowWarn = console.warn;
+      let receivedWarning: string = null;
+      const replacementWarn = (warning: string) => receivedWarning = warning;
+      console.warn = replacementWarn;
+
+      const expectedWarning = "BEEP BOOP";
+      Plottable.Utils.Window.warn(expectedWarning);
+      console.warn = originalWindowWarn;
```

Comment:
I get "`ReferenceError: sinon is not defined`" when I try to use it. Looks like maybe that's what [this pr](https://github.com/palantir/plottable/pull/2895) was trying to fix?